build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -15,11 +15,11 @@
       <HostOS>Linux</HostOS>
       <DestDir>build-tools\$(XABuildToolsVersion)</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="platform-tools_r23-linux.zip">
+    <AndroidSdkItem Include="platform-tools_r26.0.0-rc1-linux.zip">
```

Comment:
[This is failing](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/738/):

    Unable to download URL `https://dl-ssl.google.com/android/repository/build-tools_r26.0.0-rc1-macosx.zip` to `/Users/builder/android-archives/build-tools_r26.0.0-rc1-macosx.zip`: 404 (Not Found)